src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -16,7 +16,7 @@
 public class IntakeSequence extends SequentialCommandGroup {
 
   public IntakeSequence(Intake intake, Lift lift, Lights lights) {
-    addRequirements(intake, lift, lights);
+    addRequirements(intake, lift);
```

Comment:
If lights are passed into IntakeSequence and used, then why don't they need to be added as a requirement?

Reply:
I don't know why they aren't a requirement here, but I can offer a bit of extra knowledge. 

A requirement is not actually *required* to use a subsystem in a command. It essentially tells the scheduler that the command you are in is using that subsystem, and no other command can use it until the current command is finished. For example, because both the IntakeSequence and the intake default command require the intake (and IntakeSequence takes priority because it was run later, I think), the intake default command does not run while we are intaking.

Here's what WPILib says:
![image](https://user-images.githubusercontent.com/22648713/228103741-f74cb229-a488-4fe0-87fb-7e2225d4e80b.png)